x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/create.ts

```diff
@@ -53,6 +59,16 @@ export const registerSiemRuleMigrationsCreateRoute = (
 
             await ruleMigrationsClient.data.rules.create(ruleMigrations);
 
+            // Create identified resource documents without content to keep track of them
+            const resourceIdentifier = new ResourceIdentifier(firstOriginalRule.vendor);
+            const resources = resourceIdentifier
+              .fromOriginalRules(originalRules)
+              .map((resource) => ({ ...resource, migration_id: migrationId }));
+
+            if (resources.length > 0) {
```

Comment:
Can we use the `isEmpty` form `lodash` here? Not sure if it is a better option or not, just curious

```
isEmpty(resources)

Reply:
generally, if there's an easy way to do it with native code, it's usually better than using an external library